### DISCUSSION TITLE
Different rulesep in the same table

### DISCUSSION THREAD

**Discussion by Bigaluca** [lvjr/tabularray#356] [Q&A]
Is it possible to set a rulesep that only applies to the space between two specific rules? By setting the rulesep key, it seems to me that we can only define a global behavior applied to all rules.
For example I would like hlines{2} to have 6pt while hlines{3} have 3p, while any other double lines have the default space.

```latex
\documentclass{article}

\usepackage{xcolor}
\usepackage{tabularray}

\begin{document}

\begin{tblr}{
    stretch=0,
    % rulesep=5pt,
    rows={ht=2em,valign=m,mode=dmath},
    colspec={ccccc},
    hline{2} = {1}{-}{solid},
    hline{2} = {2}{-}{solid},
    hline{3} = {1}{-}{solid},
    hline{3} = {2}{-}{solid},
    hline{5} = {1}{-}{solid},
    hline{5} = {2}{-}{solid},
    hline{4,6-Z} = {solid},
    vlines = {solid},
    }
    lorem   & ipsum & ipsum & ipsum & ipsum\\
    lorem   & ipsum & ipsum & ipsum & ipsum\\
    lorem   & ipsum & ipsum & ipsum & ipsum\\
    lorem   & ipsum & ipsum & ipsum & ipsum\\
    lorem   & ipsum & ipsum & ipsum & ipsum\\
\end{tblr}

\end{document}
```

**Comment by lvjr** [comment]
You can't change `rulesep` only between two specific rules. As a workaround, you can set `rulesep=0pt` and add a white hline between them.

**Comment by Bigaluca** [comment]
Thanks for the workaround. Do you know if there is a way to insert a "transparent" hline instead of white? I tried looking at the xcolor documentation but couldn't find anything.

**Reply by lvjr** [reply]
Sorry, I don't know.
